src/components/Match/Overview/Overview.jsx

```diff
@@ -6,10 +6,13 @@ import AbilityDraftTable from '../AbilityDraftTable';
 import mcs from '../matchColumns';
 import BuildingMap from '../BuildingMap';
 import Collapsible from './../../Collapsible/index';
-import AbilityBuildTableSkeleton from './../../Skeletons/AbilityBuildTableSkeleton';
+import AbilityBuildTable from '../AbilityBuildTable';
+import deferredComponent from './../../DeferredComponent/index';
 
-const AbilityBuildTable = React.lazy(() =>
-  import(/* webpackChunkName: 'AbilityBuildTable' */ '../AbilityBuildTable'));
+
+const DeferredAbilityBuildTable = deferredComponent(AbilityBuildTable);
```

Comment:
Any reason we can't just make this a JSX component?

Reply:
There is no particular reason. I guess we could use a render prop.

Reply:
I think I prefer that pattern a little more.

Reply:
or even just this.props.children?